src/App.tsx

```diff
@@ -410,6 +415,44 @@ function App() {
                   </div>
                 </div>
 
+                {/* Collapsible section for organizer email */}
+                <CollapsibleSection
+                  initiallyOpen={!!form.creatorEmail}
+                  title={
+                    <span className="text-sm">
+                      Organizer email (optional)
```

Comment:
"Email address"

And I think we shouldn't mention "(optional)" since all fields are optional, except title, start, end and tz!
But I'm not sure yet about it.